201-customscript-extension-azure-storage-on-ubuntu/azuredeploy.parameters.json

```diff
@@ -19,6 +19,12 @@
     },
     "commandToExecute": {
       "value": "./foo.sh"
+    },
+     "customScriptStorageAccountName": {
+       "value": "GEN-UNIQUE"
+    },
+    "customScriptStorageAccountKey": {
+      "value": "GEN-UNIQUE-sakeys"
```

Comment:
I think you'll need to provide an expired sasKey, there appears to be syntax checking on the property